arches_querysets/datatypes/file.py

```diff
@@ -0,0 +1,33 @@
+from django.utils.translation import get_language
+
+from arches.app.datatypes import datatypes
+
+
+class FileListDataType(datatypes.FileListDataType):
+    localized_metadata_keys = {"altText", "attribution", "description", "title"}
```

Comment:
I still don't love the hardcoding here but I won't block. I know in certain areas of the application ( eg `card_x_node_x_widgets` , we have `"i18n_properties": ["foo"]` as part of the config. I'm hoping we can have that here as an eventual followup so that localized_metadata_keys are self-describing

Reply:
For sure https://github.com/archesproject/arches/issues/12274